decidim-blogs/spec/types/integration_schema_spec.rb

```diff
@@ -10,6 +10,8 @@
       fragment fooComponent on Blogs {
         post(id: #{post.id}) {
```

Comment:
Can we change this query so it is alphabetically (as it was before)? It's much easier to read that way

Reply:
done in [7c7ad42](https://github.com/decidim/decidim/pull/14358/commits/7c7ad42d2220e4d982785f81246a9e7418c82d76)